aura-borealis-flask-app/app.py

```diff
@@ -34,61 +37,11 @@ def get_user_selected_warnings(request):
 		collects all the checked checkboxes from a POST request used on various forms, 
 		and returns them as a list of strings
 	'''
+
 	warning_types_selected = []
-	if request.form.get('LeakingSecret') != None:
```

Comment:
Nice refactoring! You made this section so much more concise and just as understandable.